### ISSUE TITLE
About initLibNvInferPlugins

### ISSUE THREAD

**Issue by KingWangSeet** [NVIDIA/TensorRT#579]
## Description

Hi , there is question about the  initLibNvInferPlugins

## Environment

**TensorRT Version**: 6.0
**GPU Type**: 
**Nvidia Driver Version**: 
**CUDA Version**: 9.0
**CUDNN Version**: 7.6
**Operating System + Version**: 
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**: 

## Steps To Reproduce
1: In the sampleSSD.cpp:
![image](https://user-images.githubusercontent.com/23491388/83091521-ed685900-a0cd-11ea-93b4-ace0cda7ef3d.png)
2:In the plugin/InferPlugin.cpp:
![image](https://user-images.githubusercontent.com/23491388/83091892-ceb69200-a0ce-11ea-9473-5a94761ed9d7.png)
3:My Question is : When I test the initLibNvInferPlugins function in the sampleSSD.cpp, it can not into the initLibNvInferPlugins  function ; When test this:
![image](https://user-images.githubusercontent.com/23491388/83092503-083bcd00-a0d0-11ea-90d0-2a1fa790d579.png)
It also cound print anything!
In other words, i want to know the process of the lPuginCreator  registry  and the output the  PuginCreator  in the  parsers/caffe/caffeParser/caffeParser.cpp?
![image](https://user-images.githubusercontent.com/23491388/83092708-69fc3700-a0d0-11ea-8160-d9a0a4e284e4.png)

Thanks !

**Comment by KingWangSeet** [comment]
To use TensorRT registered plugins in your application, the libnvinfer_plugin.so library must be loaded and all plugins must be registered. This can be done by calling initLibNvInferPlugins(void* logger, const char* libNamespace)() in your application code.
